docs/source/reference/distributions.rst

```diff
@@ -13,12 +13,19 @@ Optuna users should not use distribution classes directly, but instead use utili
 
    FloatDistribution
    IntDistribution
+   CategoricalDistribution
+   distribution_to_json
+   json_to_distribution
+   check_distribution_compatibility
+
+The following modules are deprecated and will be removed in the future.
```

Comment:
I think "classes" is more suitable.

```suggestion
The following classes are deprecated and will be removed in the future.
```

Reply:
You are correct. Thank you for your review!